src/components/definition-list/definition-list.style.js

```diff
@@ -18,20 +18,18 @@ export const StyledDtDiv = styled.div`
   ${({ w, dtTextAlign }) => css`
     text-align: ${dtTextAlign};
     width: ${w}%;
-    white-space: nowrap;
   `}
 `;
 
 export const StyledDdDiv = styled.div`
   ${({ ddTextAlign }) => css`
     text-align: ${ddTextAlign};
-    width: auto;
-    white-space: nowrap;
+    width: -webkit-fill-available;
   `}
 `;
 
 export const StyledDt = styled.dt`
-  ${space}
```

Comment:
This is CSS just specific to your application and can't just be overwritten in Carbon like this, and `${space}` is used to add the styled-system spacing props so it can't just be taken out!